ecs_fargate/README.md

```diff
@@ -120,6 +132,19 @@ To send custom metrics by listening to DogStatsD packets from other containers,
 
 3. Finally, use a [Lambda function][14] to collect logs from CloudWatch and send them to Datadog.
 
+## Agentless Setup / Crawler-based metrics
+
+In addition to the metrics collected by the Datadog agent, we also have a CloudWatch based ECS integration. This integration collects the ECS metrics from the CloudWatch API described [here][20].
+
+As noted there, Fargate tasks also report metrics in this way:
+> The metrics made available will depend on the launch type of the tasks and services in your clusters. If you are using the Fargate launch type for your services then CPU and memory utilization metrics are provided to assist in the monitoring of your services.
+
+Since this method does not use the Datadog agent, you need to configure our AWS integration by checking 'ECS' on the integration tile. Then our application will pull in these CloudWatch metrics (namespaced `aws.ecs.*` in Datadog) on your behalf.  See the [Data Collected][21] section of the documentation.
+
+If these are the only metrics you need, you could rely on this integration for collection via CloudWatch metrics. Note though, the CloudWatch data is less granular (1-5 min depending on the type of monitoring you have enabled) and will be delayed in reporting to Datadog. This is because we must collect it from CloudWatch and adhere to their API limits, instead of pushing it to Datadog via the agent.
```

Comment:
```suggestion
If these are the only metrics you need, you could rely on this integration for collection via CloudWatch metrics. **Note**: CloudWatch data is less granular (1-5 min depending on the type of monitoring you have enabled) and delayed in reporting to Datadog. This is because we must collect it from CloudWatch and adhere to their API limits, instead of pushing it to Datadog via the Agent.
```